### ISSUE TITLE
radiomics.github.io is still accessible

### ISSUE THREAD

**Issue by ahmedhosny** [Radiomics/radiomics.github.io#31]
@JoostJM I dont have admin access, can you kill this? - its confusing I think.

**Comment by JoostJM** [comment]
You mean to delete the repository?


Joost van Griethuysen


PhD Candidate
Department of Radiology

t: (020-512)-1098
e: j.v.griethuysen@nki.nl<mailto:j.v.griethuysen@nki.nl>

[cid:image001.jpg@01D2A8A2.803019D0]

The Netherlands Cancer Institute | Plesmanlaan 121 | 1066 CX AMSTERDAM | www.nki.nl<http://www.nki.nl/>





This e-mail is intended for the addressee(s) eyes only. If you are not the intended recipient, you are hereby kindly requested to inform the sender of this. In view of the electronic nature of this communication, The Netherlands Cancer Institute (NKI) is neither liable for the proper and complete transmission of the information contained therein nor for any delay in its receipt. For information about the Netherlands Cancer Institute, go to www.nki.nl<http://www.nki.nl/>.





Dit e-mailbericht is uitsluitend bestemd voor de geadresseerde(n). Als dit bericht niet voor u bestemd is, wordt u vriendelijk verzocht dit aan de afzender te melden. Het Antoni van Leeuwenhoek  (AVL) staat door de elektronische verzending van dit bericht niet in voor de juiste en volledige overbrenging van de inhoud, noch voor tijdige ontvangst daarvan. Voor informatie over het AVL raadpleegt u www.avl.nl<http://www.avl.nl/>

**Comment by ahmedhosny** [comment]
no, go to settings > github pages > and set source to None - it might be on master now.
This way github will not publish the page at a github.io url

**Comment by JoostJM** [comment]
I can’t change this, as the repository name is radiomics.github.io (signaling it’s the repository containing github pages for the radiomics organization). I can disable the pages by renaming I think, but would you have to update links to radiomics.io?


Joost van Griethuysen


PhD Candidate
Department of Radiology

t: (020-512)-1098
e: j.v.griethuysen@nki.nl<mailto:j.v.griethuysen@nki.nl>

[cid:image001.jpg@01D2A8A3.497F70B0]

The Netherlands Cancer Institute | Plesmanlaan 121 | 1066 CX AMSTERDAM | www.nki.nl<http://www.nki.nl/>





This e-mail is intended for the addressee(s) eyes only. If you are not the intended recipient, you are hereby kindly requested to inform the sender of this. In view of the electronic nature of this communication, The Netherlands Cancer Institute (NKI) is neither liable for the proper and complete transmission of the information contained therein nor for any delay in its receipt. For information about the Netherlands Cancer Institute, go to www.nki.nl<http://www.nki.nl/>.





Dit e-mailbericht is uitsluitend bestemd voor de geadresseerde(n). Als dit bericht niet voor u bestemd is, wordt u vriendelijk verzocht dit aan de afzender te melden. Het Antoni van Leeuwenhoek  (AVL) staat door de elektronische verzending van dit bericht niet in voor de juiste en volledige overbrenging van de inhoud, noch voor tijdige ontvangst daarvan. Voor informatie over het AVL raadpleegt u www.avl.nl<http://www.avl.nl/>

**Comment by JoostJM** [comment]
On the other hand, I can specify a custom domain, is that a solution?


Joost van Griethuysen


PhD Candidate
Department of Radiology

t: (020-512)-1098
e: j.v.griethuysen@nki.nl<mailto:j.v.griethuysen@nki.nl>

[cid:image001.jpg@01D2A8A3.61B25B20]

The Netherlands Cancer Institute | Plesmanlaan 121 | 1066 CX AMSTERDAM | www.nki.nl<http://www.nki.nl/>





This e-mail is intended for the addressee(s) eyes only. If you are not the intended recipient, you are hereby kindly requested to inform the sender of this. In view of the electronic nature of this communication, The Netherlands Cancer Institute (NKI) is neither liable for the proper and complete transmission of the information contained therein nor for any delay in its receipt. For information about the Netherlands Cancer Institute, go to www.nki.nl<http://www.nki.nl/>.





Dit e-mailbericht is uitsluitend bestemd voor de geadresseerde(n). Als dit bericht niet voor u bestemd is, wordt u vriendelijk verzocht dit aan de afzender te melden. Het Antoni van Leeuwenhoek  (AVL) staat door de elektronische verzending van dit bericht niet in voor de juiste en volledige overbrenging van de inhoud, noch voor tijdige ontvangst daarvan. Voor informatie over het AVL raadpleegt u www.avl.nl<http://www.avl.nl/>

**Comment by ahmedhosny** [comment]
The setup I have now is push to the repo then I ssh into the web server (I have a clone of the repo there) then I pull the updates.
It would be great if I don't have to do that everytime I update.
could you send a screenshot of the custom domain?I have to see how it works with gandi

**Comment by JoostJM** [comment]
https://help.github.com/articles/using-a-custom-domain-with-github-pages/

[cid:image001.jpg@01D2A8A3.DCD57D00]


Joost van Griethuysen


PhD Candidate
Department of Radiology

t: (020-512)-1098
e: j.v.griethuysen@nki.nl<mailto:j.v.griethuysen@nki.nl>

[cid:image002.jpg@01D2A8A3.DCD57D00]

The Netherlands Cancer Institute | Plesmanlaan 121 | 1066 CX AMSTERDAM | www.nki.nl<http://www.nki.nl/>





This e-mail is intended for the addressee(s) eyes only. If you are not the intended recipient, you are hereby kindly requested to inform the sender of this. In view of the electronic nature of this communication, The Netherlands Cancer Institute (NKI) is neither liable for the proper and complete transmission of the information contained therein nor for any delay in its receipt. For information about the Netherlands Cancer Institute, go to www.nki.nl<http://www.nki.nl/>.





Dit e-mailbericht is uitsluitend bestemd voor de geadresseerde(n). Als dit bericht niet voor u bestemd is, wordt u vriendelijk verzocht dit aan de afzender te melden. Het Antoni van Leeuwenhoek  (AVL) staat door de elektronische verzending van dit bericht niet in voor de juiste en volledige overbrenging van de inhoud, noch voor tijdige ontvangst daarvan. Voor informatie over het AVL raadpleegt u www.avl.nl<http://www.avl.nl/>

**Comment by ahmedhosny** [comment]
akh ya that wont work coz the website will be hosted on github and just redirecting from the custom domain to it. We wont have the SSL certificates that we have now with it being hosted on gandi.
If we change the name, would it mess up the clones? or it is smart enough?
I can always reclone it
